desktop.css

```diff
@@ -483,3 +483,113 @@ nav {
   margin-bottom: 24px;
   border-color: #dfe1e6;
 }
+
+/* pop up */
+.popUp {
+  width: 950px;
+  height: 600px;
+  padding: 24px;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+.pop-up-desktop-1 {
+  display: grid;
+  grid-template-columns: 1.6fr 1fr;
+  gap: 25px;
+  width: 130vh;
+  height: 55vh;
+}
+
+.pop-up-desktop-2 {
+  display: flex;
+  flex-direction: column;
+  justify-content: start;
+  margin-left: -50px;
+}
+
+.pop-up-image {
+  margin-top: 0;
+  margin-bottom: 15px;
+  height: 90%;
+}
+
+.pop-up-h1 {
+  font-family: "Crete Round", serif;
+  font-size: 32px;
+  font-weight: 400;
+  line-height: 44px;
+  letter-spacing: 0;
+  text-align: left;
+  color: #172b4d;
+}
+
+.close-icon-2 {
+  position: absolute;
+  top: 25px;
+  right: 20px;
+  width: 20px;
+  height: 20px;
+  background: #eee;
+  color: #111;
+  border: none;
+  outline: none;
+  border-radius: 50%;
+  cursor: pointer;
+}
+
+/* body.active-popUp {
+  overflow: hidden;
+}
+
+body.active-popUp .main {
+  filter: blur(5px);
+  background: rgba(0, 0, 0,0.08);
+  transition: filter 0ms ease-in-out 0ms;
+}
+
+body.active-popUp .popUp {
+  top: 50%;
+  opacity: 1;
+  margin-top: 0px;
+  transition: top 0ms ease-in-out 0ms,
+                  opacity 300ms ease-in-out,
+                  margin-top 300ms ease-in-out;
+} */
```

Comment:
- [ ] Please remove the commented code from your `file` because it is unprofessional to have some commented-out code in your files, where comments are meant for adding some sort of information which will help the user to understand your code but seems like this commented code is confusing and creates so many questions related to code quality. Kindly remove it.